packages/block-library/src/table/deprecated.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText, getColorClassName } from '@wordpress/block-editor';
+
+const blockAttributes = {
```

Comment:
We're only adding a wrapper element right? Can't we share this attribute definition between files?

Reply:
Yes, thanks for pointing that out. Done!